shuffle/cli/api.md

```diff
@@ -0,0 +1,143 @@
+# Shuffle API
+
+All commands are shown like so: `generic_command` | `example_command`
+
+## shuffle new
+
+**Usage:**
+
+`shuffle new <project_path>` | `shuffle new ~/Desktop/TestCoin`
+
+Takes in a project_path and creates a new shuffle project at that location with all pre-generated folders which can be used down the line. Also creates ~/.shuffle/Networks.toml which is your networks configuration file.
+
+Note: Feel free to add any network to connect to with the same format as the localhost network (already in the file) to Networks.toml.
+
+## shuffle node
+
+**Usage:**
+
+`shuffle node`
+
+Runs a local node at endpoint http://127.0.0.1:8080 which listens to all transactions sent by the user. Also creates ~/.shuffle/nodeconfig directory which contains information to configure the local node.
```

Comment:
Is there documentation we can point to on how one can generate their own genesis bundle?

Reply:
Yes there is a separate genesis tutorial. Maybe we should link it here https://github.com/diem/diem/blob/main/shuffle/cli/tutorials/Genesis.md